theano/tensor/nnet/tests/test_abstract_conv.py

```diff
@@ -770,6 +770,10 @@ def tcase(self, i, f, s, b, flip, provide_shape, fd=(1, 1)):
                            mode=mode)
 
     def tcase_gi(self, i, f, o, s, b, flip, provide_shape, fd=(1, 1), expect_error=False):
+
+        if not theano.config.cxx:
+            raise SkipTest("Need cxx to test conv2d")
```

Comment:
This one is supposed to pass, as it is testing the Python perform in AbstractConv2d.
It is possible that some calls to verify_grad are failing because _they_ insert nodes that require a c++ compiler.